FaceID/DB_faceID.py

```diff
@@ -296,17 +300,38 @@ def searchFace():
 
             for(x,y,w,h) in faces: # detects face
 
-                EmpID, confidence = recognizer.predict(gray[y:y+h,x:x+w]) # recognizes face
-              
-                # Checks confidence threshold  
-                if ((confidence < 100) and (100-confidence) > 25):
+                EmpID, accuracy = recognizer.predict(gray[y:y+h,x:x+w]) # recognizes face
+                
+                accuracy = round(100 - accuracy)
+                    
+                if (accuracy <= 100) and (accuracy > 0):
```

Comment:
We can simplify this by writing: `if (0 < accuracy <= 100)`

Reply:
Fixed it!